src/main/java/picocli/CommandLine.java

```diff
@@ -13924,7 +13934,7 @@ public void run() {
             if (parent == null) { return; }
             Help.ColorScheme colors = colorScheme != null ? colorScheme : Help.defaultColorScheme(ansi);
             if (commands.length > 0) {
-                CommandLine subcommand = parent.getSubcommands().get(commands[0]);
+                CommandLine subcommand = parent.getCommandSpec().subcommands().get(commands[0]);
```

Comment:
To avoid unnecessary copies here

Reply:
What did you think of my [suggestion](https://github.com/remkop/picocli/issues/1156#issuecomment-690960531) (which may have been a bit hidden in the code) to use 

```
AbbreviationMatcher.match(parent.getCommandSpec().subcommands().keySet(),
                        commands[0], parent.getCommandSpec().subcommandsCaseInsensitive(), self);
```

to also add support for abbreviations to the `HelpCommand`?

Reply:
Oops, nevermind, looks like you already did this.